scripts/generate_genotype_df/generate_genotype_df.py

```diff
@@ -29,15 +30,22 @@ def query():
     mt = mt.annotate_rows(freq=ht[mt.row_key].freq)
     mt = mt.filter_rows(mt.freq.AF[1] > 0.01)
     # select alleles and locus (contig and position must be selected separately),
-    # then convert to pandas
-    t = mt.rows()
+    t = mt.entries()
+    # get genotype codes (n_alt_alleles, as in PLINK files)
+    t = t.annotate(n_alt_alleles=t.GT.n_alt_alleles())
     t = t.key_by(contig=t.locus.contig, position=t.locus.position)
-    t = t.select(t.alleles)
-    pd = t.to_pandas(flatten=True)
-    # expand locus to two columns and rename 
+    t = t.select(t.alleles, t.s, t.n_alt_alleles)
+    df = t.to_pandas(flatten=True)
     # save each chromosome to an individual file
-    for chr in set(pd['contig']): 
-        pd.loc[pd['contig'] == chr].to_parquet(output_path(f'tob_genotype_maf01_{chr}.parquet'))
+    for chr in set(df['contig']):
+        df = df.loc[df['contig'] == chr]
+        # create a new matrix with SNPs as columns and samples as rows
+        # with genotype values as entries
+        df['snp_id'] = df['contig'].map(str) + ':' + df['position'].map(str) + ':' + df['alleles'].str[0] + ':' + df['alleles'].str[1]
```

Comment:
Didn't we think recently that SNP ID isn't needed? Why not export these individual fields as different columns?

Reply:
I'm trying to get a matrix which holds genotype information for each sample across all SNPs. The SNP ID wouldn't be needed if you were only looking at allele information, but across genotype information, it's the only way to have a row x column matrix. e.g., 

| Sample  | SNP1 | SNP2 | SNP3 |
|---------| ------|-------|------|
| sample1 | 1  | 0 | 0 | 1 |
| sample2 | 2  | 1 | 2 | 1 |
| sample3 | 0  | 0 | 2 | 1 |